src/Discord/Parts/Embed/Embed.php

```diff
@@ -38,7 +38,104 @@ class Embed extends Part
      * {@inheritdoc}
      */
     protected $fillable = ['title', 'type', 'description', 'url', 'timestamp', 'color', 'footer', 'image', 'thumbnail', 'video', 'provider', 'author', 'fields'];
-
+    
+    /**
+     * Constructs a new instance.
+     * @param array  $embed
+     * @throws \Throwable
+     */
+    function __construct(array $embed = array()) {
```

Comment:
Constructor needs to extend the `Part` constructor otherwise no factory, http etc.